src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string;
+  text: string;
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => (
+  <NavLink
+    to={to}
+    className={({ isActive }) => classNames(
+      'navbar-item', { 'is-active': isActive },
+    )}
+    style={({ isActive }) => ({ color: isActive ? 'red' : 'black' })}
```

Comment:
it's bad practice to write inline styles, better to take a color from 'is-active' class